config/cmake/H5pubconf.h.in

```diff
@@ -108,7 +112,7 @@
 #cmakedefine H5_HAVE_CODESTACK @H5_HAVE_CODESTACK@
 
 /* Define to 1 if you have the <curl/curl.h> header file. */
-#cmakedefine H5_HAVE_CURL_H @H5_HAVE_CURL_H@
+#cmakedefine H5_HAVE_CURL_CURL_H @H5_HAVE_CURL_H@
```

Comment:
H5_HAVE_CURL_CURL_H is not used in the library. It's just an artifact of the Autotools configure checks.